README.md

```diff
@@ -9,8 +9,11 @@ Using your AWS API credentials, S3Proxy allows you to make unauthenticated HTTP
 
 This makes it easy for tools like curl to access private buckets:
 
+Eg. Assuming you have a S3 bucket named `bucket1` and it has the following 
+structure `folder1/folder2/item.txt`. 
+
 ```bash
-curl http://localhost:9090/object/key
+curl http://localhost:9090/bucket1/folder1/folder2/item.txt?aws_region=eu-west-1
```

Comment:
So would I be able to hit the _eu-west-1_ endpoint to retrieve an object from a bucket in _us-east-1_? If that is the case, I would rather configure the region in the options. Because being a request parameter implies that it is optional. And if the global s3 endpoint doesn't work at the moment, it will result in a poor user experience.